CMakeLists.txt

```diff
@@ -728,6 +728,7 @@ set(SOURCES
         env/fs_remap.cc
         env/mock_env.cc
         env/unique_id_gen.cc
+        env/flink/jni_helper.cc
```

Comment:
Should "env/flink/env_flink.cc" be placed near this position as well?

Reply:
Thanks for the reminder. 
Moved.